src/App/ImagePlane.h

```diff
@@ -43,6 +43,11 @@ class AppExport ImagePlane : public App::GeoFeature
     App::PropertyLength       XSize;
     App::PropertyLength       YSize;
 
+    double getXSize();
+    double getYSize();

```

Comment:
Both methods should be declared `const`. But actually all the new methods are superfluous because they do nothing than directly accessing the property to get or set the value.

Reply:
Function removed.